frontend/src/components/fileUploader.module.css

```diff
@@ -0,0 +1,90 @@
+.uploadButton_container {
+  position: relative;
+  display: flex;
+  align-items: flex-start;
+  justify-content: center;
+  background-color: transparent;
+  border-radius: 50%;
+  height: 36px;
+  width: 36px;
+  margin-right: 16px;
+  margin-left: 10px;
+  margin-top: 5px;
+}
+
+.uploadButton {
+  background: transparent;
+  border: none;
+  height: 36px;
+  width: 36px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+}
+
+.uploadButton img {
+  width: 24px;
+  height: 24px;
+}
+
+.uploadButton:not(:disabled):hover {
+  cursor: pointer;
+}
+
+.uploadButton:active {
+  background-color: var(--grey-400);
```

Comment:
I noticed when clicking the button the background has a square shape. I think that can be an easy fix by adding` border-radius: 50%;` here.

Reply:
Done :)